packages/indexer/src/indexers/program-idl-data-indexer.ts

```diff
@@ -0,0 +1,60 @@
+import { Context, PublicKey } from "@solana/web3.js";
+import { Result, TaggedUnion } from "../match";
+
+export type DaoIndexer = {
```

Comment:
I think we should just use `IntervalFetchIndexer` for these likely.

Reply:
OR we can use an AccountInfoIndexer to listen for changes on like the autocrat account as an alternative.

Reply:
lmk if  you have questions on this